src/components/MainPage.js

```diff
@@ -66,7 +71,7 @@ class MainPage extends Component<Props, State> {
       return <LinearProgress color="secondary" variant="query" />;
     } else {
       return (
-        <div>
+        <main style={{ padding: 20 }}>

```

Comment:
In general, I am against inline styles. Can you reuse something from `material` here? Or just use CSS.